airflow/ui/src/pages/DagsList/DagsFilters.tsx

```diff
@@ -133,7 +142,21 @@ export const DagsFilters = () => {
           </Select>
         </Box>
       </HStack>
-      <ReactSelect isDisabled placeholder="Filter by tag" />
+      <ReactSelect
+        aria-label="Filter DAGs by tag"
+        classNamePrefix="react-select"
```

Comment:
```suggestion
        chakraStyles={{
          container: (provided) => ({
            ...provided,
            minWidth: 64,
          }),
        }}
```

We don't need the classNamePrefix but we can add a minWidth.